src/drivers/driver_zephyr.c

```diff
@@ -236,6 +237,444 @@ static void wpa_drv_zep_event_proc_unprot_disassoc(struct zep_drv_if_ctx *if_ctx
 			event);
 }
 
+struct phy_info_arg {
+	u16 *num_modes;
+	struct hostapd_hw_modes *modes;
+	int last_mode, last_chan_idx;
+	int failed;
+	u8 dfs_domain;
+};
+
+static void phy_info_freq_cfg(struct hostapd_hw_modes *mode,
+		struct hostapd_channel_data *chan,
+		struct wpa_supp_event_channel *chnl_info)
+{
+	u8 channel = 0;
+
+	chan->freq = chnl_info->center_frequency;
+	chan->flag = 0;
+	chan->allowed_bw = ~0;
+	chan->dfs_cac_ms = 0;
+	
+	if (ieee80211_freq_to_chan(chan->freq, &channel) != NUM_HOSTAPD_MODES) {
+		chan->chan = channel;
+	}
+	if (chnl_info->wpa_supp_flags & WPA_SUPP_CHAN_FLAG_FREQUENCY_DISABLED)
+		chan->flag |= HOSTAPD_CHAN_DISABLED;
```

Comment:
Mix and match of braces for single if-condition, we should stick to either Zephyr (on) or Linux (off).

Reply:
Its better to stick with supplicant formatting since we would have to upstream at some point.

Reply:
Yes, in that case we can remove braces for single if.